public/static/docs/command-reference/install.md

```diff
@@ -16,6 +16,9 @@ like Git to store code and configuration files. With `dvc install`, the two are
 more tightly integrated in order to cause certain convenient actions to happen
 automatically.
 
+> Note that the hooks only have an effect in repository versions (branches,
```

Comment:
see the suggestion above, and probably this document is the only place we should keep this note. Also not sure if we need to make it `>` ... may be keep as a regular paragraph?

Reply:
OK, using

```markdown
Note that this command requires the <abbr>DVC project</abbr> to be a Git
repository. But **the hooks won't activate** if the current branch (commit, tag,
etc.) doesn't have DVC initialized (no `.dvc/` directory present).
```

(not in block quote)

Pushed in 014e57f8.